apps/nextjs-website/src/components/organisms/WebinarDetailTemplate/WebinarDetailTemplate.tsx

```diff
@@ -166,6 +168,35 @@ const WebinarDetailTemplate = ({ webinar }: WebinarDetailTemplateProps) => {
       {bodyContent}
       {speakerList}
       {startInfo}
+      {webinar.relatedResources && (
+        <RelatedResources
+          title={webinar.relatedResources.title}
+          resources={webinar.relatedResources.resources.map((resource) => ({
+            title: resource.title,
+            description: {
+              title: resource.subtitle || '',
+              content: resource.description,
+            },
+            imagePath: resource.image?.url || '/images/hero.jpg',
+            link: {
+              label: resource.linkText,
+              href: resource.linkHref,
+            },
+            mobileImagePath: resource.image?.url || '/images/hero.jpg',
+          }))}
+          downloadableDocuments={(
+            webinar.relatedResources.downloadableDocuments || []
+          ).map((doc) => ({
+            title: doc.title,
+            size: doc.size,
+            downloadLink: doc.downloadLink,
+            tags: [{ label: doc.extension }],
+          }))}
+        />
+      )}
+      {webinar.questionsAndAnswers && (
+        <QuestionsAndAnswers items={[...webinar.questionsAndAnswers]} />
+      )}
```

Comment:
I think you should move `QuestionsAndAnswers` before `RelatedResources`
![image](https://github.com/user-attachments/assets/7c5bc709-bd35-4a10-b7b4-b06591f10270)